src/lib/rmt/wizard_scc_page.rb

```diff
@@ -126,14 +128,27 @@ def scc_credentials_valid?
       )
     )
 
-    uri = URI('https://scc.suse.com/connect/organizations/systems')
+    uri = URI('https://scc.suse.com/connect/organizations/orders')
     req = Net::HTTP::Get.new(uri)
     req.basic_auth(@config['scc']['username'], @config['scc']['password'])
-
-    res = Net::HTTP.start(uri.host, uri.port, use_ssl: true) { |http| http.request(req) }
+    req['User-Agent'] = YAST_RMT_USER_AGENT
+
+    valid_credentials = nil
+    while valid_credentials.nil?
+      begin
+        res = Net::HTTP.start(uri.host, uri.port, use_ssl: true) { |http| http.request(req) }
```

Comment:
Just a minor suggestion:

```suggestion
        res = Net::HTTP.start(uri.host, uri.port, use_ssl: uri.is_a?(URI::HTTPS)) { |http| http.request(req) }
```

This is more robust as it will still work after you change the URL above to plain HTTP.

Reply:
It's a nice enhancement, but unless the URL is changed in the code, it will not be very helpful.

I wonder if it would be valuable for test and development purposes to have a way to configure the SCC URL (e.g.: env var), then this change would be very welcome. I had to manually change both the URL and the `use_ssl` flag whenever I tested it locally.